Source/NaturalLanguageClassifierV1/NaturalLanguageClassifier.swift

```diff
@@ -0,0 +1,239 @@
+        }
+        
+        // construct REST request
+        let request = RestRequest(
+            method: .POST,
+            url: serviceURL + "/v1/classifiers/\(classifierId)/classify",
+            acceptType: "application/json",
+            contentType: "application/json",
+            messageBody: body
+        )
+        
+        // execute REST request
+        Alamofire.request(request)
+            .authenticate(user: username, password: password)
+            .responseObject(dataToError: dataToError) {
+                (response: Response<Classification, NSError>) in
+                switch response.result {
+                case .Success(let classification): success(classification)
+                case .Failure(let error): failure?(error)
+                }
+            }
+    }
+    
+    /**
+     Deletes the classifier with the classifierId
+     
+     - parameter classifierId:      The classifer ID used to delete the classifier
+     - parameter failure: A function executed if an error occurs.
+     - parameter success: A function executed with the list of available standard and custom models.
+     */
+    public func deleteClassifier(
+        classifierId: String,
+        failure:(NSError -> Void)? = nil,
+        success: (Void -> Void)? = nil) {
+        
+        // construct REST request
+        let request = RestRequest(
+            method: .DELETE,
+            url: serviceURL + "/v1/classifiers/\(classifierId)",
+            acceptType: "application/json"
+        )
+        
+        // execute REST request
+        Alamofire.request(request)
+            .authenticate(user: username, password: password)
+            .responseData { response in
+                switch response.result {
+                case .Success(let data):
+                    switch self.dataToError(data) {
+                    case .Some(let error): failure?(error)
+                    case .None: success?()
+                    }
+                case .Failure(let error):
+                    failure?(error)
+                }
+        }
+    }
+
+    /**
+     Returns the status and other information about a classifier
+     
+     - parameter classifierId: The classifer ID used to retrieve the classifier
+     - parameter failure: A function executed if an error occurs.
+     - parameter success: A function executed with the list of available standard and custom models.
+     */
+    public func getClassifier(
+        classifierId: String,
+        failure: (NSError -> Void)? = nil,
+        success: ClassifierDetails -> Void) {
+        
+        // construct REST request
+        let request = RestRequest(
+            method: .GET,
+            url: serviceURL + "/v1/classifiers/\(classifierId)",
+            acceptType: "application/json"
+        )
+        
+        // execute REST request
+        Alamofire.request(request).authenticate(user: username, password: password).responseObject(dataToError: dataToError) {
```

Comment:
This line can be split up into several lines to maintain consistency with other methods.